raiden/ui/app.py

```diff
@@ -247,6 +248,27 @@ def run_app(
     known_given_network_id = given_network_id in ID_TO_NETWORKNAME
     known_node_network_id = node_network_id in ID_TO_NETWORKNAME
 
+    if config['services']['pathfinding_service_address'] is None:
+        click.secho(
+            "There is no pathfinding service defined, basic routing will be used",
+        )
+    else:
+        pathfinding_service_info = get_pfs_info(config['services']['pathfinding_service_address'])
+        if pathfinding_service_info is False:
```

Comment:
Nitpick: `if not pathfinding_service_info`

Reply:
changed